classes/Admin/Popups.php

```diff
@@ -1213,6 +1246,27 @@ public static function render_columns( $column_name, $post_id ) {
 				case 'popup_title':
 					echo esc_html( $popup->get_title() );
 					break;
+				case 'enabled':
+					if ( 'publish' === $post->post_status ) {
+						$enabled = $popup->get_meta( 'enabled' );
+						if ( '' === $enabled ) {
+							$enabled = 1;
+						} else {
+							$enabled = intval( $enabled );
+						}
+						$nonce = wp_create_nonce( "pum_save_enabled_state_{$popup->ID}" );
+						?>
+						<div class="pum-toggle-button">
+							<input id="pum-enabled-toggle-<?php echo esc_attr( $popup->ID ); ?>" type="checkbox" <?php checked( 1, $enabled ); ?> class="pum-enabled-toggle-button" data-nonce="<?php echo esc_attr( $nonce ); ?>" data-popup-id="<?php echo esc_attr( $popup->ID ); ?>">
```

Comment:
Missing `value="1"`, not sure you need it or not, but it may get flagged as invalid without it.

Reply:
@danieliser `value` attribute is not required as checkboxes default to a value of `on`.

![image](https://user-images.githubusercontent.com/9730040/93469941-2f0c5200-f8bf-11ea-9643-68ef77311679.png)

I just ran our HTML of that page through a validator just in case and, while there are several other errors (mostly from WP core surprisingly), there is no error relating to these checkboxes.

Reply:
Sweet, learn something new.